MapboxNavigation/NavigationViewController.swift

```diff
@@ -369,6 +371,10 @@ public class NavigationViewController: NavigationPulleyViewController, RouteMapV
             }
         }
         
+        if route != routeController.routeProgress.route {
```

Comment:
Is this check safe when we don't implement `-hash`, `-isEqual:` or an overloaded operator?